doc/source/whatsnew/v0.21.0.txt

```diff
@@ -25,6 +25,35 @@ New features
 - Added ``__fspath__`` method to :class:`~pandas.HDFStore`, :class:`~pandas.ExcelFile`,
   and :class:`~pandas.ExcelWriter` to work properly with the file system path protocol (:issue:`13823`)
 
+
+``infer_objects`` type conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``DataFrame`` and ``Series`` have gained an ``infer_objects`` method to perform dtype inference
+on object columns, replacing some of the functionality of the deprecated ``convert_objects``
```

Comment:
maybe want to make these `` :func:`Series.infer_objects` `` (and the like), but needs rewording to do that